src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -135,6 +143,9 @@ private[query] class EventsByTagPublisher(
 
     case Cancel =>
       context.stop(self)
+
+    case tagWritten: String if tagWritten == tag =>
+      self ! Continue
```

Comment:
I'm thinking if we need to take the consistency-delay into account? This will not do much good when a normal consistency-delay is used.